pytorch_lightning/utilities/cli.py

```diff
@@ -155,7 +155,11 @@ def add_lightning_class_args(
             if subclass_mode:
                 return self.add_subclass_arguments(lightning_class, nested_key, fail_untyped=False, required=required)
             return self.add_class_arguments(
-                lightning_class, nested_key, fail_untyped=False, instantiate=not issubclass(lightning_class, Trainer)
+                lightning_class,
+                nested_key,
+                fail_untyped=False,
+                instantiate=not issubclass(lightning_class, Trainer),
+                sub_configs=True,
```

Comment:
Does this need to bump the min jsonargparse version?

Reply:
No.